scripts/jenkins/ardana/ansible/files/rpm-added-files-whitelist.txt

```diff
@@ -0,0 +1,28 @@
+# alternatives system is expected to change stuff under /usr
+-> /etc/alternatives/.
+
+# Installed by osconfig/ansible/role/udev/tasks/configure.yml
+/usr/lib/systemd/system/ardana-udev\.service
```

Comment:
do we need to escape? so this is some regex file?

Reply:
Yes, it's used by `grep`.